.github/workflows/pages.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: Pages
+
+on:
+  workflow_run:
+    workflows: [Release]
+    branches: [release]
+    types: [completed]
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: pages
+  cancel-in-progress: true
+
+jobs:
+  build:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: release
```

Comment:
@JoseLion since there is a branch filter on top for the `release` branch, I wonder why we need to hardcode the ref here?

Reply:
I think workflows with `workflow_run` always run on the main branch. But even if that's the case, that should not be an issue for the GH pages site. I'll remove it, thanks! 🙂